deployments/agent-rest/docker-compose.yml

```diff
@@ -105,4 +105,4 @@ services:
 
 networks:
   agent_sdk_net:
-    driver: bridge
+    external: true
```

Comment:
can you move docker-compose under test/bdd

Reply:
similar to other projects

Reply:
For now we are not using it for BDD test. We use it for demo (swagger spec). I will move it when we add project-related  BDD tests. Make sense? I just noticed that `network` should be fixed and I did it.